lib/actions/user.js

```diff
@@ -152,6 +152,25 @@ export function createOrUpdateUser (userData, silentOnSuccess = false) {
   }
 }
 
+/**
+ * Requests the verification email for the new user to be resent.
+ */
+export function resendVerificationEmail (auth0) {
+  return async function (dispatch, getState) {
+    const { apiBaseUrl, apiKey } = getMiddlewareVariables(getState())
+    const accessToken = await auth0.getAccessTokenSilently()
```

Comment:
Isn't accessToken stored in the reducer now? See line 123 above.

Reply:
`accessToken` is not stored yet because it is set that the same time as the user, and the user didn't get set yet. Should that get split?

Reply:
Hmm, I'm not sure. Perhaps we should split the set access token functionality out from set logged in user. What do you think about having an additional check in `UserLoaderScreen` that checks if `accessToken` is available from the store? If not, it can just call `auth0.getAccessTokenSilently()` and store it. That would allow us to then fetch or initialize user without passing `auth0`.

```javascript
// UserLoaderScreen
componentDidUpdate () {
    const { accessToken, auth0, fetchOrInitializeUser, setAccessToken } = this.props
    if (!accessToken) {
      let token = await auth0.getAccessTokenSilently()
      setAccessToken({accessToken: token, auth0User: auth0.user})
    }
    if (accessToken && this.loggedInUserIsUnfetched()) {
      fetchOrInitializeUser() // can now get accessToken and auth0User from store
    }
}
```

Reply:
@landonreed let me think what you think of this: https://github.com/opentripplanner/otp-react-redux/pull/227/commits/b9b0ba35ab20c854ca8c40b9229c79ab2b85f092. We still need the `auth0` argument to initialize a default user (needs the auth0 id and email).

Reply:
That looks good to me. But `resendVerificationEmail` no longer needs the auth0 param now, right?

Reply:
Good catch. Updated in https://github.com/opentripplanner/otp-react-redux/pull/227/commits/77a1a2c0f422a76331c75fbaea09579946a51692.